config/passport.js

```diff
@@ -0,0 +1,50 @@
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+const User = require('../models/user')
```

Comment:
No semi-colon on this line, add it to be consistent... or you could remove all the semi colons, ES5 doesn't actually need them at all.